WalletWasabi/Blockchain/WalletVirtualOutput.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using WalletWasabi.Blockchain.TransactionOutputs;
+using WalletWasabi.Blockchain.Keys;
+
+namespace WalletWasabi.Blockchain;
+
+public record WalletVirtualInput
```

Comment:
The file name doesn't match the class name.

Reply:
Fixed in https://github.com/zkSNACKs/WalletWasabi/pull/7853/commits/91d4e679598d1a9219c9c1ea42ec163f5780cbff.